sriovnet.go

```diff
@@ -382,3 +383,23 @@ func FreeVfByNetdevName(handle *PfNetdevHandle, vfIndex int) error {
 func GetVfNetdevName(handle *PfNetdevHandle, vf *VfObj) string {
 	return vfNetdevNameFromParent(handle.PfNetdevName, vf.Index)
 }
+
+// GetVfIndexByPciAddress gets a VF PCI address (e.g '0000:03:00.4') and
+// returns the correlate VF index.
+func GetVfIndexByPciAddress(vfPciAddress string) (string, error) {
+	vfPath := filepath.Join(PciSysDir, vfPciAddress, "physfn/virtfn*")
+	matches, err := filepath.Glob(vfPath)
+	if err != nil {
+		return "", err
+	}
+	for _, match := range matches {
+		tmp, err := os.Readlink(match)
+		if err != nil {
+			continue
+		}
+		if strings.Contains(tmp, vfPciAddress) {
+			return virtFnRe.FindStringSubmatch(match)[1], nil
```

Comment:
@moshe010 did I miss virtFnRe definition?
I didn't' notice in previous patch or this one.